src/lang/std/sketch.ts

```diff
@@ -811,6 +811,7 @@ export const tangentialArcTo: SketchLineHelper = {
     }
     const newLine = createCallExpression('tangentialArcTo', [
       createArrayExpression([toX, toY]),
+      false,
```

Comment:
This is a real problem.

```suggestion
      createLiteral(false),
```